python-avd/pyavd/_eos_designs/schema/schema_fragments/fabric_ip_addressing.schema.yml

```diff
@@ -23,7 +23,7 @@ keys:
                   This allocation method will skip every other /31¹ subnet making it less space efficient than `odd_id`.
                 - odd_id: `(odd_id - 1) / 2`. Requires the node_group to have a node with an odd ID and a node with an even ID.
                 - same_subnet: the offset will always be zero.
-                  This allocation method will cause every MLAG link to be addressed with the same /31¹ subnet.
+                  This allocation method will use the first available /<prefixlen> subnet from the provided pool for all MLAG links.
```

Comment:
prefixlen may mot be coming from the pool - just use the same footnote right?

Prefix length is used from there:

        prefixlen = (
            self.inputs.fabric_ip_addressing.mlag.ipv6_prefix_length if address_family == "ipv6" else self.inputs.fabric_ip_addressing.mlag.ipv4_prefix_length
        )

Reply:
```suggestion
                  This allocation method will use the first /31¹ subnet from the pool for all MLAG links.
```